lms/djangoapps/shoppingcart/models.py

```diff
@@ -1939,12 +1941,17 @@ def additional_instruction_text(self):
                 "If you haven't verified your identity yet, please start the verification process ({verification_url})."
             ).format(verification_url=verification_url)
 
+        if is_professional_mode_verified:
+            refund_reminder_msg = _("You have only until two days after the course starts to unenroll and receive a "
```

Comment:
Should this refund message match the language of the original message?
`You have up to two days into the course to unenroll and receive a full refund.`

/CC @srpearce